packages/path_provider/path_provider_linux/pubspec.yaml

```diff
@@ -16,12 +16,12 @@ flutter:
         dartPluginClass: PathProviderLinux
 
 dependencies:
-  ffi: ">=1.1.2 <3.0.0"
+  ffi: ">=2.0.1"
   flutter:
     sdk: flutter
-  path: ^1.8.0
-  path_provider_platform_interface: ^2.0.0
-  xdg_directories: ">=0.2.0 <2.0.0"
+  path: ^1.8.2
+  path_provider_platform_interface: ^2.0.6
+  xdg_directories: ">=1.0.0"
```

Comment:
This removes compatibility with version 0.2.0 for no reason (the APIs are identical), creating more potential conflicts in the ecosystem.